src/Symfony/Component/HttpFoundation/Cookie.php

```diff
@@ -38,10 +39,11 @@ class Cookie
      * @param bool                          $secure   Whether the cookie should only be transmitted over a secure HTTPS connection from the client
      * @param bool                          $httpOnly Whether the cookie will be made accessible only through the HTTP protocol
      * @param bool                          $raw      Whether the cookie value should be sent with no url encoding
+     * @param bool|string                   $sameSite Whether the cookie will be available for cross-site requests
```

Comment:
``string|null`` would be better than ``string|bool`` IMO (especially given that ``true`` is not a valid value for this argument)